src/client/directory.ts

```diff
@@ -10,60 +10,150 @@
  */
 
 /**
- * Type definitions that is are wherever the API expects the ID or of an application to be passed. We have both, since
- * internally appId is used to identify app, but externally we use name to repsect specification
+ * Type alias to indicate when an Application Identifier should be passed. Application Identifiers
+ * are described [here](https://fdc3.finos.org/docs/1.0/appd-discovery#application-identifier).
  *
- * It is not possible for TypeScript to verify that only valid application names/appIds are passed
- * to the FDC3 API. This type definition is more a hint to the callee, over a plain "string" argument.
+ * This type alias exists to disambiguate raw string app identity from the [[AppName]].
```

Comment:
I don't feel like this is true? Like, if we had 3 or 0 other AppFoo strings the sentence about this one still holds and we'd still want this

Though if we keep this, should it be: "This type alias exists to disambiguate *the* raw string app identity from the [[AppName]]."?

Reply:
So I feel like explaining how typescript works should not be a feature of our docs. So I removed that. Happy to return it if that;s the consensus.

Reply:
Ah, yep, totally agree on "let's not explain Typescript in our docs" 🙂